mpo.rga.Dayon.yml

```diff
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home:rw
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - tar -xf dayon.tgz
+      - install dayon/dayon.jar /app
+      - install dayon/dayon_*.sh /app
+      - install dayon.sh /app
+      - install dayon.browser /app
+      - install -d /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 dayon/dayon.png /app/share/icons/hicolor/128x128/apps/mpo.rga.Dayon.png
+      - install -Dm644 dayon.assisted.desktop /app/share/applications/mpo.rga.Dayon.assisted.desktop
```

Comment:
this is not the right filename. It should match the `app-id`. (ONE of them)

Reply:
Hm, the app consists of two parts: `assistant` (the one offering help) and the `assisted` (the one needing help).
Both functionalities are in the same jar and are distributed as one packet.

Reply:
it doesn't matter. There must be one .desktop with the right app-id based name.